src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+package com.denizenscript.depenizen.bukkit.commands.quests;
+
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+
+import me.blackvein.quests.Quest;
+import me.blackvein.quests.Quester;
+import me.blackvein.quests.Quests;
+import me.blackvein.quests.exceptions.InvalidStageException;
+import me.blackvein.quests.util.Lang;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+
+import org.bukkit.entity.Player;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.depenizen.bukkit.bridges.QuestsBridge;
+
+/**
+ * Command integration for Quests
+ * @author PikaMug
+ * @since 2.0.0-b595
+ */
+public class QuestsCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name Quests
+    // @Syntax quests [add/remove/set] (questId:<quest-id>) (stage-no:<#>) (points:<#>) (state:true/false)
+    // @Group Depenizen
+    // @Plugin Depenizen, Quests
+    // @Required 1
+    // @Short Edits quest player information.
+    //
+    // @Description
+    // This command allows you to give, quit, or set the stage of a quest for a player, or to add, subtract, or set
+    // the amount of Quest Points that a player holds.
+    //

```

Comment:
Please define here what the input arguments do. At least what `state` is, as that's not particularly obvious (the others seem to be more or less clear enough, though rule of thumb is more explanation = better!)

Reply:
I used the mcMMO command for reference which also does not say what `state` is. The usage gives an idea of its purpose, but I'll be glad to mention it explicitly.

Reply:
Do not use Depenizen as a reference.